ipn/ipnserver/server.go

```diff
@@ -61,6 +61,9 @@ type Options struct {
 	// StatePath is the path to the stored agent state.
 	StatePath string
 
+	// KubeSecret is the name of the kubernetes secret to use.
```

Comment:
// KubeSecret, if not empty, is the ....

And Kubernetes capitalized.

And what's it used for? And what's the relation between this and StatePath?